fsspec/implementations/tests/test_common.py

```diff
@@ -0,0 +1,33 @@
+import datetime
+import time
+import pytest
+
+from fsspec import AbstractFileSystem
+from fsspec.implementations.tests.conftest import READ_ONLY_FILESYSTEMS
+
+TEST_FILE = "file"
```

Comment:
Will this create a (temporary) file in the local directory? May be better using `tempfile.mktemp`